Homework_2.sql

```diff
@@ -0,0 +1,33 @@
+USE baseball;
```

Comment:
Minor comment: The code runs on an assumption that the baseball tarball is downloaded and the baseball database is already created. It could be modified to download the tar ball dynamically and create the database, without making the tarball a part of it.